platform/viewer/src/components/Header/Header.js

```diff
@@ -60,6 +60,7 @@ class Header extends Component {
         onClick: () =>
           show(ConnectedUserPreferencesForm, {
             title: t('User Preferences'),
+            t,
```

Comment:
Let's avoid passing translation methods via props, we can use hooks or higher-order components available.